Sources/Tracking/CustomerIO.swift

```diff
@@ -35,16 +35,17 @@ public extension CustomerIO {
         configure configureHandler: ((inout SdkConfig) -> Void)?
     ) {
         var sdkConfig = SdkConfig.Factory.create(siteId: siteId, apiKey: apiKey, region: region)
+        let newDiGraph = DIGraph(sdkConfig: sdkConfig)
+        let implementation = DataPipeline.initialize(moduleConfig: DataPipelineConfigOptions.Factory.create(sdkConfig: sdkConfig))
+        let migrationAssistant = newDiGraph.dataPipelineMigrationAssistant
 
         if let configureHandler = configureHandler {
             configureHandler(&sdkConfig)
         }
 
-        let implementation = DataPipeline.initialize(moduleConfig: DataPipelineConfigOptions.Factory.create(sdkConfig: sdkConfig))
-
+        // Handle logged-in user from Journeys to CDP
+        migrationAssistant.handleAlreadyIdentifiedMigratedUser()
```

Comment:
I'm okay with this, but I would have preferred making only 1 call to `migrationAssistant` here, where it could further break things inside the class and do what it wants to do. e.g.

```swift
migrationAssistant.handleMigration()
```

and inside

```swift
func handleMigration() {
    handleAlreadyIdentifiedMigratedUser()
    handleQueueBacklog()
}
```

Reply:
Thanks for the suggestion, find changes [here](https://github.com/customerio/customerio-ios/pull/490/commits/05ddf03e2b45379c67de1cac6347930e7618e6b6).